### DISCUSSION TITLE
Install with HomeBrew (See https://brew.sh/)

### DISCUSSION THREAD

**Discussion by johnrm9** [aleppo-io/last-window-quits#19] [Feature requests]
Please put Last Window Quit on HomeBrew. See https://brew.sh/
To install:
brew install --cask last-window-quits

**Comment by lawand-dot-io** [comment]
Good idea, will look into it. Thanks

**Comment by lawand-dot-io** [comment]
@johnrm9 the app is now added to homebrew: https://formulae.brew.sh/cask/last-window-quits

Cheers
